common/app/Panes/Panes.jsx

```diff
@@ -53,29 +55,53 @@ export class Panes extends PureComponent {
       render,
       panes
     } = this.props;
-    return panes.map(({ name, left, right, dividerLeft }) => {
-      const divider = dividerLeft ?
-        (
-          <Divider
-            key={ name + 'divider' }
-            left={ dividerLeft }
-            name={ name }
-          />
-        ) :
-        null;
+    if (panes.length > 0) {
+      return panes.map(({ name, left, right, dividerLeft }) => {
+        const divider = dividerLeft ?
+          (
+            <Divider
+              key={ name + 'divider' }
+              left={ dividerLeft }
+              name={ name }
+            />
+          ) :
+          null;
 
-      return [
-        <Pane
-          key={ name }
-          left={ left }
-          right={ right }
-          >
-          { render(name) }
-        </Pane>,
-        divider
-      ];
-    }).reduce((panes, pane) => panes.concat(pane), [])
-      .filter(Boolean);
+        return [
+          <Pane
+            key={ name }
+            left={ left }
+            right={ right }
+            >
+            { render(name) }
+          </Pane>,
+          divider
+        ];
+      }).reduce((panes, pane) => panes.concat(pane), [])
+        .filter(Boolean);
+    } else {
+      return this.renderQuote(quoteObj);
+    }
+  }
+
+  renderQuote() {
```

Comment:
You are passing in the `quoteObj` as an argument on line 83, would it be better to access it here as a param instead of referencing the top level `quoteObj`?

Reply:
Ah, sorry! I missed that. Forgot to change that back after I was trying some ways to change the quote when the panes were closed and open, rather than when the window is resized. I believe that referencing the top level quoteObj would be better here.